enhancements/insights/pulling-sca-certs-from-ocm.md

```diff
@@ -0,0 +1,155 @@
+---
+title: pulling-and-exposing-sca-certs-from-ocm
+authors:
+  - "@tremes"
+reviewers:
+  - "@sbose78"
+  - "@inecas"
+  - "@petli-openshift"
+  - "@bparees"
+  - "@dhellman"
+  - "@mfojtik"
+  - "@adambkaplan"
+approvers:
+  - "@sbose78"
+  - "@bparees"
+  - "@dhellman"
+creation-date: 2021-03-04
+last-updated: 2021-08-10
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Insights Operator pulling and exposing SCA certs from the OCM API
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement will enable the Insights Operator to pull the data (SCA certs)
+from the OCM (OpenShift Cluster Manager) API. The data will be exposed by the Insights Operator
+in the OpenShift API to allow users to use them when consuming and building container images
+on the platform.
+
+## Motivation
+
+Users could consume RHEL content and container images using the RHEL subscription in the OpenShift 3.x.
+In the OpenShift 4, this is no longer possible because the Red Hat Enterprise Linux Core OS (RHCOS) does not
+provide any attached subscription. This enhancement is to provide users the Simple Content Access (SCA) certs
+from Red Hat Subscription Manager (RHSM).
+
+### Goals
+
+- Extend the Insights Operator config with an OCM API URL to be able to query the data
+- Periodically pull the data from the OCM API and expose it in the OpenShift API
+- This is an opt-in feature by a cluster user and might be moved to a different OCP component in the future
```

Comment:
If we think this might move to a different component in the future, should the config be tied to the insights operator configuration now?

Reply:
I added new section `Configuration` explaining the current situation https://github.com/openshift/enhancements/pull/683/files#diff-93dffd2aa59c973a916306675f62907275cff29ff9a16ba77aaf98cfa830249fR102